content_dev/docs/sdk/java/getting-started/instance/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: codelab
+
+title: Creating an instance of AtClient # Step Name
+description: | # SEO Description for this step
+  Create an instance of AtClient with one of the various factory methods
+
+draft: false # TODO CHANGE THIS TO FALSE WHEN YOU ARE READY TO PUBLISH THE PAGE
+order: 1 # Ordering of the steps
+---
+
+To create an instance of AtClient, use one of the factory methods. Note: you must have the `.atKeys` file in a `keys/` directory. You can generate a .atKeys file from using the Register CLI or Onboaring CLI if you already own the atSign.
```

Comment:
Just to note that we need to make a change which will look for .atKeys files in (1) a keys/ subdirectory or (2) `homedir/.atsign/keys`. And ideally we should be able to provide a path-to-keys-file. Would you like to pick up this task and then reflect it in the docs, @JeremyTubongbanua ?

Reply:
Hmm this is definitely something to think about.